webapp-django/crashstats/signature/static/signature/js/signature_tab_aggregations.js

```diff
@@ -45,7 +45,8 @@ SignatureReport.AggregationsTab.prototype.loadControls = function() {
     // Set the placeholder.
     this.$selectElement.select2({
         'placeholder': 'Aggregate on...',
-        'allowClear': true
+        'allowClear': true,
+        'sortResults': socorro.search.sortResults,
```

Comment:
This is unrelated, but while I was refactoring the ``search.js`` file, I also fixed a bug that's been bothering me for some time. Now all field inputs in Signature report are sorted correctly!